Documentation/configuration.md

```diff
@@ -1,34 +1,36 @@
-## Configuration
+# Configuration
 
 flannel reads its configuration from etcd.
-By default, it will read the configuration from `/coreos.com/network/config` (can be overridden via `--etcd-prefix`).
-You can use `etcdctl` utility to set values in etcd.
+
+By default, it will read the configuration from `/coreos.com/network/config` (which can be overridden using `--etcd-prefix`).
+
+Use the `etcdctl` utility to set values in etcd.
+
 The value of the config is a JSON dictionary with the following keys:
 
-* `Network` (string): IPv4 network in CIDR format to use for the entire flannel network.
-This is the only mandatory key.
+* `Network` (string): IPv4 network in CIDR format to use for the entire flannel network. (This is the only mandatory key.)
 
 * `SubnetLen` (integer): The size of the subnet allocated to each host.
-   Defaults to 24 (i.e. /24) unless the Network was configured to be smaller than a /24 in which case it is one less than the network.
+   Defaults to 24 (i.e. /24) unless `Network` was configured to be smaller than a /24 in which case it is one less than the network.
 
 * `SubnetMin` (string): The beginning of IP range which the subnet allocation should start with.
-   Defaults to the first subnet of Network.
+   Defaults to the first subnet of `Network`.
 
 * `SubnetMax` (string): The end of the IP range at which the subnet allocation should end with.
-   Defaults to the last subnet of Network.
+   Defaults to the last subnet of `Network`.
 
 * `Backend` (dictionary): Type of backend to use and specific configurations for that backend.
    The list of available backends and the keys that can be put into the this dictionary are listed below.
-   Defaults to "udp" backend.
+   Defaults to `udp` backend.
```

Comment:
And I don't know why, but I cannot reply to your previous comment. The etcd comment must have been original to the doc. Where can I find info on setting up k8s config?